scripts/fast-start.sh

```diff
@@ -137,6 +137,7 @@ if [ ! -e AppScaleFile ]; then
         echo "admin_user : $ADMIN_EMAIL" >> AppScalefile
         echo "admin_pass : $ADMIN_PASSWD" >> AppScalefile
     fi
+    echo "group : faststart-${PROVIDER}" >> AppScalefile
```

Comment:
This will make highly likely that one will be in the same group (just use faststart twice with the same provider). Have you tested if there is any sideeffect to try it twice? Or to try while another faststart instance is already running? Good work! :+1: